app/assets/stylesheets/pygments.scss

```diff
@@ -63,3 +63,33 @@
 .highlight .ld, /* Literal.Date */
 .highlight .m   /* Literal.Number */
 { color: #ca1655; }
+
+.highlight {
+  position: relative;
+
+  &:focus, &:hover {
+    .highlight__button {
+      visibility: visible;
+    }
+  }
+}
+
+.highlight__button {
+  visibility: hidden;
+  position: absolute;
+  right: .8em;
```

Comment:
Let's use `right: .4em;` and `top: .4em;` here, as there are rare occasions one-line code blocks.

<img width="789" alt="Screen Shot 2023-01-09 at 11 22 54 pm" src="https://user-images.githubusercontent.com/7202667/211307277-e2e952f5-36ad-41ea-94b5-e725ac82dc68.png">